### PR TITLE
Move product build out of fragments & into coursebook

### LINKED ISSUE #455: move self selected project files into coursebook & change "week-UX" to be "design sprint" inside
[This proposal](https://github.com/foundersandcoders/london-curriculum/issues/7) was passed, to turn weeks 10-12 into a 3 week product build.

In the "product build" folder we can replace
> [Design sprint. To be completed by @harrygfox @nogainbar @bradreeder](https://github.com/foundersandcoders/master-reference/tree/master/coursebook/fragments/product-build#day-1-5)

with the content of the ["week UX" readme](https://github.com/foundersandcoders/master-reference/tree/master/coursebook/week-UX/README.md)

### PR DESCRIPTION
**Pull request by jsms90**
+ move `product build` files out from `fragments` and name this "weeks 10-12"
+ move "UX week" into "design sprint" within `weeks-10-12`
+ remove learning outcomes link from UX week, as this is already in `weeks-10-12/README.md`
+ move `tech-choices.md` into new "build sprint" folder
+ move day 15 schedule breakdown from `product-build/README.md` into `build-sprint/README.md`
+ add description of day 6-14
  + day 6: students pitch their own project ideas
  + day 7-14: link to tech choices doc

Fixes #455 

Relates #500 #88 

Review requested from @mattlub @m4v15 for their own awareness.

Reviews required before merging: @sofer @des-des @bradreeder @eliascodes

### CODE REVIEW

File: coursebook/weeks-10-12/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Product Build
+
+## Menu
+ - [Learning outcomes](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/fragments/product-build/learning-outcomes)
```
Comment (eliascodes) [inline coursebook/weeks-10-12/README.md]: I think these links need to be updated, right?

Comment (jsms90) [inline coursebook/weeks-10-12/README.md]: Fixed

File: coursebook/weeks-10-12/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Product Build
+
+## Menu
+ - [Learning outcomes](./learning-outcomes)
```
Comment (eliascodes) [inline coursebook/weeks-10-12/README.md]: This needs a `.md` extension. At the moment it's coming back with 404

File: coursebook/README.md [hunk 3]
```diff
@@ -17,3 +15,6 @@ related to each week see:
 * [Week 6](week-6/README.md) - PostgreSQL
 * Week 7 - Authentication
 * Week 8 - Express
+
+The second half of the curriculum is focused on longer projects and introduces you to client interaction:
++ [Weeks 10-12](./week-10-12) - self-selected project
```
Comment (sofer) [inline coursebook/README.md]: Links should be `./weeks-10-12`

File: coursebook/weeks-10-12/README.md (hunk 2 above)
Comment (jsms90) [inline coursebook/weeks-10-12/README.md]: :+1: done

File: coursebook/README.md (hunk 3 above)
Comment (jsms90) [inline coursebook/README.md]: :+1: done

**Comment by jsms90** [comment]
Thanks for catching these @eliascodes @sofer !! Changes made :+1:

**Pull request merged**